tuxemon/core/states/combat/combat_menus.py

```diff
@@ -147,7 +147,10 @@ def choose_technique():
                 if tech.next_use <= 0:
                     image = self.shadow_text(tech.name)
                 else:
+                    tmp = self.font_color
+                    self.font_color = (220, 220, 220)
                     image = self.shadow_text("%s -%s" % (tech.name, tech.next_use))
```

Comment:
I'm not a huge fan of this. `shadow_text` already has an optional kwarg for the background color, lets add one for the foreground color.  If the foreground color is passed, use that, if not, use `self.color`.  We should move this gray color with the rest so that colors are consistent.  https://github.com/Tuxemon/Tuxemon/blob/development/tuxemon/core/menu/menu.py#L51

Reply:
I agree with that

Reply:
@bitcraft you should check:
5d5824e

Reply:
looks perfect